wfl/Database/MongoDBManager.cs

```diff
@@ -26,6 +26,19 @@ public MongoDBManager()
             await collection.InsertOneAsync(r);
         }
 
+        public async Task<bool> CheckRestaurantIdAsync(string id)
```

Comment:
Why do we need to check a restaurant id?  One key thing to think through when writing code is to simplify your interfaces.  If you can use the same interface for 2 purposes, then you absolutely should do it.  It'll make your API simpler to consume and re-use.  So in this case, could we instead use the Get method, and when it returns null you know there is no item at that id?  Then we can offer a single interface and still solve all use cases.